requirements/install.pip

```diff
@@ -1,2 +1,2 @@
-udata>=2.0.0
-influxdb==5.2.3
+udata>=6.0.0
+udata-front@git+https://github.com/maudetes/udata-front@feat/use-metrics-data
```

Comment:
maybe update that ? at least keep it in mind that it refers to a non-merged and published branch